### DISCUSSION TITLE
this started popping up today, "TypeError: Cannot read properties of undefined (reading 'length')"

### DISCUSSION THREAD

**Discussion by ghost** [rfennell/AzurePipelines#1484] [Q&A]
I cant provide a data dump as its a GCC system, but this just started today.

**Comment by ghost** [comment]
this i can share, error content from within ADO:

{"Major":"4","Minor":"5","Patch":"2"},"issueType":"error","errorMessage":"{\"stack\":\"TypeError: Cannot read properties of undefined (reading 'length')\\n    at Object.<anonymous> (D:\\\\a\\\\_tasks\\\\XplatGenerateReleaseNotes_7b0a5887-75c4-4ade-a915-f92a93c934cb\\\\4.5.2\\\\ReleaseNotesFunctions.js:1785:84)\\n    at Generator.next (<anonymous>)\\n    at fulfilled (D:\\\\a\\\\_tasks\\\\XplatGenerateReleaseNotes_7b0a5887-75c4-4ade-a915-f92a93c934cb\\\\4.5.2\\\\ReleaseNotesFunctions.js:5:58)\\n    at runMicrotasks (<anonymous>)\\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\",\"message\":\"Cannot read properties of undefined (reading 'length')\"}"}

**Reply by rfennell** [reply]
I introduced a bug in the last release, it only appears when a pipeline publishes no artifacts. The fix will be out in the next hour

It is being tracked in #1485

**Reply by rfennell** [reply]
Fix was released as 4.6.2

**Reply by ghost** [reply]
just tested today, looks great, thank you for that.  As I have said before, this is a great tool!
